README.md

```diff
@@ -1243,3 +1243,78 @@ out  | evt.ota_end.report      | object     | Sent on upgrade end with upgrade s
    }
 }
 ```
+
+### Zwave Specific service 
+This service is used to meet all zwave specific requirements that are not satisfied with already implemented services.
+
+#### Service name 
+
+`zwave_specific`
+
+#### Interfaces
+
+Type | Interface               | Value type | Description 
+-----|-------------------------|------------|--------------
+out  | evt.notification.report | str_map    | Notification Reports that are sent by slave devices.
+
+#### Storage property
+
+Key        | Value description 
+-----------|----------------------------------
+sub_value  | Values pair to store in Vinculum.
+
+#### Examples
+
+Motion Detection occurs for state Motion Sensor Status:
```

Comment:
I would use the value names in the header here to make it easier to read the example, e.g.

"`motion_detection` detected for state `motion_sensor_status`.".